### ISSUE TITLE
Client: Browser locks up when player joins or host attempts to start game

### ISSUE THREAD

**Issue by bneff84** [Lattyware/massivedecks#118]
Seems there's a bug in the client code that's putting the browser into an infinite loop which causes an "unresponsive" warning and prevents starting the game.

I've reproduced with a deployment of the master branch to Heroku myself. I've also reproduced using https://md.rereadgames.com

Steps:
1. Create a new game
2. Add an AI or have a player join using the URL (or open it yourself via a new browser/session)
3. Client will freeze for the host

I'm pretty unfamiliar with the project itself and with ELM unfortunately so I didn't dig too much deeper than this.

**Comment by bneff84** [comment]
Since I left the windows open while writing up this issue, I can say the browser does eventually catch up and the user will join. It took several minutes on my machine. This has been reproduced in FireFox 74.0.1 and Chrome 80.0.3987.149 in both normal and incognito modes.

**Comment by Lattyware** [comment]
Yeah, I can reproduce this. Not quite sure what is going on, I'll take a look.

**Comment by Lattyware** [comment]
I can't reproduce this locally, so I'm pushing a tiny change to see if this resolves itself with a new build. It might be some weirdness from the JS optimisation (which would explain why I don't see it running a development version).

If it doesn't, I'll do some further local testing.

**Comment by Lattyware** [comment]
For now, I have rolled back to a previous build (`65bdb389100e4411e1886158ed48c9a4ce77b377-dev`) on the hosted instance, so it should work but unfortunately the latest changes (mainly the new translation) are naturally not there. As far as I can tell this isn't actually an issue with those changes, but something in the build process.

**Comment by nohit** [comment]
I was seeing it on the rereadgames server and reproed it with heroku.  Games are spinning and timing out before we can even start them. :(

**Comment by Lattyware** [comment]
@nohit It should be temporarily worked around now on the hosted version, I can't reproduce it now I rolled back to a previous build. If you are running your own instance, use commit 65bdb389100e4411e1886158ed48c9a4ce77b377 for now (that's the one that works).

**Comment by Lattyware** [comment]
OK, I just pushed a new version that changes how the minification is done and hopefully fixes it—my local docker build ran fine, so seems good. I'm going to test it on a heroku build and then I'll set the hosted instance back to using latest if it is good.

**Comment by Lattyware** [comment]
OK, latest version is now live and appears to work, I can't reproduce on any version any more including the hosted instance. Please reopen this or create a new issue if you run into further problems.

Thanks for the bug report.

**Comment by nohit** [comment]
Thanks. Fix verified.
